src/app/fdctl/config.c

```diff
@@ -570,6 +571,18 @@ fdctl_cfg_from_env( int *      pargc,
 
     config->tiles.net.ip_addr = iface_ip;
     mac_address( config->tiles.net.interface, config->tiles.net.mac_addr );
+
+    /* support for multihomed hosts */
```

Comment:
Can you check that none of the multi-homed IP addresses are the same here, and also that none of them are the same as the iface_ip we calculated above. (I think the net tile will fail with an opaque error if these are true).